deploy/kiali/kiali_cr.yaml

```diff
@@ -492,22 +498,37 @@ spec:
 #      url: ""
 #
 # **Istio-specific settings:
+# component_status:
+#   enabled: Enable/Disable of istio component status into masthead indicator. It defaults to true.
+#   components: A list of components that Kiali will check its statuses.
+#     app_label: Istio component pod app label.
+#     is_core: Whether the component is core for your deployment.
 # istio_identity_domain: The annotation used by Istio to identify domains.
 # istio_sidecar_annotation: The pod annotation used by Istio to identify the sidecar.
-# istio_status_enabled: Enable/Disable of istio component status into masthead indicator. It defaults to true.
 # url_service_version: The Istio service used to determine the Istio version. If empty, assumes the URL for the well-known Istio version endpoint.
 #    ---
 #    istio:
+#      component_status:
+#        enabled: true
+#        components:
+#          - app_label: istiod
+#            is_core: true
+#          - app_label: istio-ingressgateway
+#            is_core: true
+#          - app_label: istio-egressgateway
+#            is_core: false
```

Comment:
sorry if I missed this earlier... in every yaml used in the operator everywhere, we never indent the list. Keep it at the same level as the parent (this keeps the lists from growing too wide as you get deeper in the hierarchy).

```suggestion
#        components:
#        - app_label: istiod
#          is_core: true
#        - app_label: istio-ingressgateway
#          is_core: true
#        - app_label: istio-egressgateway
#          is_core: false
```

Reply:
Didn't realized. good catch. fixed in my latest commit.